app/javascript/angular/dashboard/widget-wrapper.js

```diff
@@ -68,11 +87,11 @@ ManageIQ.angular.app.component('widgetWrapper', {
         </div>
         <widget-error ng-if="vm.error === true"></widget-error>
         <widget-spinner ng-if="!vm.widgetModel && vm.widgetBlank == 'false' && !vm.error"></widget-spinner>
-        <div ng-if="vm.widgetBlank === 'true' || vm.widgetModel"
+        <div ng-if="vm.widgetModel.blank === 'true' || vm.widgetModel"
              ng-attr-id="{{vm.innerDivId}}"
              ng-class="{ hidden: vm.widgetModel.minimized, mc:true }">
-          <widget-empty ng-if="vm.widgetBlank === 'true'"></widget-empty>
-          <div ng-if="vm.widgetBlank === 'false'"
+          <widget-empty ng-if="vm.widgetModel.blank === 'true'"></widget-empty>
```

Comment:
`! vm.widgetModel || vm.widgetModel.blank === 'true'` ?

Reply:
For error there's `vm.error` and specific component `widget-error`.